examples/BrokerServerTestCase/CMakeLists.txt

```diff
@@ -14,13 +14,10 @@ target_link_libraries(brokerServerTestCase PUBLIC HELICS::apps)
 set_target_properties(brokerServerTestCase PROPERTIES FOLDER examples)
 
 if(HELICS_BUILD_TESTS AND NOT MSVC)
-   # add_test(
-    #    NAME brokerServerTestCase
-    #    COMMAND
-    #        ${PROJECT_SOURCE_DIR}/scripts/test_example.sh 10
-    #        $<TARGET_FILE:nonlings_fed1_cpp> $<TARGET_FILE:nonlings_fed2_cpp>
-    #)
-    #set_property(TEST nonlings_example PROPERTY LABELS Continuous)
+    # add_test(
+    # NAME brokerServerTestCase COMMAND ${PROJECT_SOURCE_DIR}/scripts/test_example.sh 10
+    # $<TARGET_FILE:nonlings_fed1_cpp> $<TARGET_FILE:nonlings_fed2_cpp> ) set_property(TEST
+    # nonlings_example PROPERTY LABELS Continuous)
```

Comment:
@nightlark  is this whole if clause something we need?

Reply:
I think we can get rid of it.